### DISCUSSION TITLE
Issues with Data Augmentation

### DISCUSSION THREAD

**Discussion by rachyuthan** [tudelft/AE4353-Y24#26] [Ex_2]
Hello,

I included code for the data augmentation to flip both the vertical and horizontal axes but when I set the augmentation to True and then run it in the visualization I do not see that a full representation on the polar plot. Images for further context:

![image](https://github.com/user-attachments/assets/b8a219d0-e28d-455d-8616-1d70a110cb27)

![image](https://github.com/user-attachments/assets/d535a9c0-7c7b-4b69-8708-ee7803dacfb3)

I want to make sure that I have this correct before running the training loop so should I be seeing the results of the data augmentation already or is it only visible once the training loop has been executed.

Thanks

**Comment by yilun-wu** [comment]
Perhaps you are plotting the distribution the `train_dataset`. Note the augmentation is performed in the dataloader. The `dataloader` takes the data from `train_dataset` (raw unaugmented) and then augments it and return the augmented version in the training loop to feed it to the model. If you plot the distribution within the training loop, you should see a difference.
